src/index.js

```diff
@@ -2,5 +2,5 @@
 
 export {default} from './command'
 export {default as Topic} from './topic'
-export {BooleanFlag, StringFlag, NumberFlag, Flag} from './flags'
+export {boolean, string, number, app, remote, Flag} from './flags'
```

Comment:
so you don't want `const flags = require('cli-engine-command')` as your import statement, rather `const flags = require('cli-engine-command').flags`?